Gemfile

```diff
@@ -57,6 +57,7 @@ gem 'paperclip',                      "~>4.3.0"
 
 # Not vendored and not required
 gem "ancestry",                       "~>2.1.0",      :require => false
+gem "azure-armrest",                                  :path => "/Users/bronaghsorota/dev/azure-armrest"
```

Comment:
@chessbyte
yea that is on my list of outstanding items above (which I added as an amendment so you probably didn't see it).